cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -1207,6 +1208,336 @@ Rolling up the above using ``rollup=workflow`` would group the programs together
+   * - ``prefix``
+     - Optional ``prefix``, when provided only fields that have given prefix will be returned
+
+|
+
+Following is sample response::
+
+  ["firstName","lastName","item","id","customer_id"]
+
+.. highlight:: console
+
+.. rubric:: HTTP Responses
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Status Codes
+     - Description
+   * - ``200 OK``
+     - Fields of dataset are returned as a list of strings in the body of the response
+   * - ``400 BAD REQUEST``
+     - Failure to parse the time range provided
+
+Field Lineage Summary
+---------------------
+
+Get the field lineage summary for a specified field of a dataset. Field lineage summary consists of set of
+dataset and fields belonging to them which participated in the computation of a given field of a dataset::
+
+  GET /namespaces/{namespace-id}/datasets/{dataset-id}/lineage/fields/{field-name}?start=<start-ts>&end=<end-ts>&direction=incoming
+
+where:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``namespace-id``
+     - Namespace ID
+   * - ``dataset-id``
+     - Name of the ``dataset``
+   * - ``field-name``
+     - Name of the ``field``
+   * - ``start-ts``
+     - Starting time-stamp (inclusive), in seconds. Supports ``now``, ``now-1h``, etc. syntax.
+   * - ``end-ts``
+     - Ending time-stamp (exclusive), in seconds. Supports ``now``, ``now-1h``, etc. syntax.
+   * - ``direction``
+     - ``incoming``, to return the set of dataset and fields which participated in the computation of the given field
+
+|
+
+Returned response consists of direction in which the summary is requested (currently only ``incoming`` direction is
+supported) along with the list of datasets and fields which were responsible for computation of the given field.
+
+Following is sample response(pretty-printed)::
+
+  {
+      "incoming": [
+          {
+              "dataset": {
+                  "dataset": "Customer",
+                  "entity": "DATASET",
+                  "namespace": "default"
+              },
+              "fields": [
+                  "body"
+              ]
+          },
+          {
+              "dataset": {
+                  "dataset": "purchases",
+                  "entity": "DATASET",
+                  "namespace": "default"
+              },
+              "fields": [
+                  "body"
+              ]
+          }
+      ]
+  }
+
+.. highlight:: console
+
+.. rubric:: HTTP Responses
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Status Codes
+     - Description
+   * - ``200 OK``
+     - Fields of dataset are returned as a list of strings in the body of the response
+   * - ``400 BAD REQUEST``
+     - Failure to parse the time range provided
+
+Field Lineage Operations
+------------------------
+
+Get the details of operations responsible for computation of a specified field of a dataset over a given time range::
```

Comment:
Gets the details of operations responsible for computation of a specified field of a dataset for a specified range of time.

Reply:
fixed.